internal/pkg/describe/status.go

```diff
@@ -280,24 +304,38 @@ func (s *ecsServiceStatus) HumanString() string {
 	writer.Flush()
 	fmt.Fprintf(writer, "  %s\t%s\n", "Updated At", humanizeTime(s.Service.LastDeploymentAt))
 	fmt.Fprintf(writer, "  %s\t%s\n", "Task Definition", s.Service.TaskDefinition)
+
 	fmt.Fprint(writer, color.Bold.Sprint("\nTask Status\n\n"))
 	writer.Flush()
 	headers := []string{"ID", "Image Digest", "Last Status", "Started At", "Capacity Provider", "Health Status"}
 	fmt.Fprintf(writer, "  %s\n", strings.Join(headers, "\t"))
 	fmt.Fprintf(writer, "  %s\n", strings.Join(underline(headers), "\t"))
 	for _, task := range s.Tasks {
-		fmt.Fprint(writer, task.HumanString())
+		fmt.Fprintf(writer, "  %s\n", task.HumanString())
 	}
+
 	if len(s.StoppedTasks) > 0 {
 		fmt.Fprint(writer, color.Bold.Sprint("\nStopped Tasks\n\n"))
 		writer.Flush()
 		headers = []string{"ID", "Image Digest", "Last Status", "Started At", "Stopped At", "Stopped Reason"}
 		fmt.Fprintf(writer, "  %s\n", strings.Join(headers, "\t"))
 		fmt.Fprintf(writer, "  %s\n", strings.Join(underline(headers), "\t"))
 		for _, task := range s.StoppedTasks {
-			fmt.Fprint(writer, (awsECS.StoppedTaskStatus)(task).HumanString())
+			fmt.Fprintf(writer, "  %s\n", (awsECS.StoppedTaskStatus)(task).HumanString())
 		}
 	}
+
+	if len(s.TasksTargetHealth) > 0 {
```

Comment:
I wonder if our output is going to be too long vertically and result in poor signal/noise ratio.   
I'd really like to keep all the information in a single table if possible.  

Here are some wild 🔥 ideas:
1. We introduce a "summary" table.  
The stacked bar can be colored (where the boxes are green, yellow, red for health, and blue and magenta for Fargate and Spot)
```
Task Summary
   Status                    ■■■■■■■■■□      Running (9/10)     Stopped (1/10)
   Container Health (CH)     □□□□□□□□□□      Unknown (10/10)
   HTTP Health (HH)          ■■■■■■■■■■      Healthy (10/10)  
   Image Digest              ■■■■■■□□□□      e52e8d6f (6/10)    123hfaa7 (4/10)
   Capacity Providers (CP)   ■■■■■■□□□□      Fargate (6/10)     Spot (4/10)
```
2. If there are any stopped tasks, then we follow the summary with where each column width is limited:
```
Stopped tasks
   Reason                Count        Task IDs
   Healthcheck failed    3            0b6fca20,28719a2c,46b4b163
```
3. Finally, we display the tasks at the bottom but we don't have to display all the information and reduce the gap between columns
```
Tasks
  ID          Status   Digest    Started At  CH        HH        CP 
  0b6fca20    Running  e52e8d6f  1 week ago  Unknown   Healthy   Fargate
  28719a2c    Running  e52e8d6f  1 week ago  Unknown   Healthy   Fargate
  46b4b163    Running  e52e8d6f  1 week ago  Unknown   Healthy   Fargate
  5615eb22    Running  123hfaa7  1 min ago   Unknown   Healthy   Spot
  6aa4075e    Running  123hfaa7  1 min ago   Unknown   Healthy   Spot
```

Putting it all together we get:
```
Task Summary
   Status                    ■■■■■■■■■□      Running (9/10)     Stopped (1/10)
   Container Health (CH)     □□□□□□□□□□      Unknown (10/10)
   HTTP Health (HH)          ■■■■■■■■■■      Healthy (10/10)  
   Image Digest              ■■■■■■□□□□      e52e8d6f (6/10)    123hfaa7 (4/10)
   Capacity Providers (CP)   ■■■■■■□□□□      Fargate (6/10)     Spot (4/10)

Stopped tasks
   Reason                Count        Task IDs
   Healthcheck failed    3            0b6fca20,28719a2c,46b4b163

Tasks
  ID          Status   Digest    Started At  CH        HH        CP 
  0b6fca20    Running  e52e8d6f  1 week ago  Unknown   Healthy   Fargate
  28719a2c    Running  e52e8d6f  1 week ago  Unknown   Healthy   Fargate
  46b4b163    Running  e52e8d6f  1 week ago  Unknown   Healthy   Fargate
  5615eb22    Running  123hfaa7  1 min ago   Unknown   Healthy   Spot
  6aa4075e    Running  123hfaa7  1 min ago   Unknown   Healthy   Spot
```

Reply:
I'll just throw in another task summary take - i haven't thought through this at all really. The main thrust is keeping the cool bars indicative of some type of progress - like, we want to get all tasks to running, or all health checks to passing, or a deployment to the latest. 

For capacity - i don't know if the cool bars make sense, so I just changed it to f and s. 

Anyway, just an idea!

```
Task Summary
   Running                   ■■■■■■■■■□      9/10 Desired Tasks Running     
   Healthy                   ■■■■■■■■■■      10/10 Passing HTTP Health Checks
   Service Deployment        ■■■■■■□□□□      6/10 on e52e8d6f (latest), 4/10 on 123hfaa7
   Capacity                  ffffffffss      8/10 on Fargate 2/10 on Fargate Spot

Tasks
  ID          Status   Digest    Started At   Health    Capacity 
  0b6fca20    Running  e52e8d6f  1 week ago   Healthy   Fargate
  28719a2c    Running  e52e8d6f  1 week ago   Healthy   Fargate
  46b4b163    Running  e52e8d6f  1 week ago   Healthy   Fargate
  5615eb22    Running  123hfaa7  1 min ago    Healthy   Spot
  6aa4075e    Running  123hfaa7  1 min ago    Healthy   Spot
```